### PR TITLE
Final/verified mentor check

### PR DESCRIPTION
**Pull request by shaargtz**
# What does this PR do?
- Adds a message indicating that whoever posted a question or answer is a verified mentor.

**Related Issues**
- NA

---
**Merge Checklist**
- [x] Branch is up to date
- [x] No code style issues

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -513,7 +513,11 @@ function createAnswerElement(answer) {
   // answersElement.appendChild(votesElement);
   
   const authorElement = document.createElement('small');
-  authorElement.innerText = answer.authorName;
+  if (answer.isVerifiedMentor) {
+    authorElement.innerText = answer.authorName + '\nVerified Ex-Intern';
+  } else {
+    authorElement.innerText = answer.authorName;
+  }
```
Comment (OUMontiel) [inline src/main/webapp/script.js:520]: Wouldn't it be better to do this:
```
  authorElement.innerText = answer.authorName;
  if (answer.isVerifiedMentor) {
    authorElement.innerText += '\nVerified Ex-Intern';
  }
```

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -536,7 +540,11 @@ function createCommentElement(comment) {
   commentElement.innerText = comment.body;
   
   const authorElement = document.createElement('small');
-  authorElement.innerText = comment.authorName;
+  if (comment.isVerifiedMentor) {
+    authorElement.innerText = comment.authorName + '\nVerified Ex-Intern';
+  } else {
+    authorElement.innerText = comment.authorName;
+  }
```
Comment (OUMontiel) [inline src/main/webapp/script.js:547]: Same here.

File: src/main/webapp/script.js (hunk 1 above)
Comment (shaargtz) [reply src/main/webapp/script.js]: done

File: src/main/webapp/script.js (hunk 2 above)
Comment (shaargtz) [reply src/main/webapp/script.js]: done

**Pull request merged**